R/data.R

```diff
@@ -208,6 +208,20 @@
 #' @family anchored datasets
 "centered_ipd_twt"
 
+#' Weighted object for two arm trial data
+#' @format A `maicplus_estimate_weights` object created by [estimate_weights()] containing
```

Comment:
Could you explain which parameters were used in estimate_weights() to generate this object?

Reply:
what do you mean..
estimate_weights(data = centered_ipd_twt, centered_colnames = centered_colnames, n_boot_iteration = 1000)

these are three parameters used..

Reply:
Can you include that in the roxygen documentation?

Reply:
#' Weighted object for two arm trial data. **Parameters data, centered_colnames, and n_boot_iteration was used.**
#' @format A `maicplus_estimate_weights` object created by [estimate_weights()] containing
#'   \describe{
#'     \item{data}{patient level data with weights}
#'     \item{centered_colnames}{Columns used in MAIC}
#'     \item{nr_missing}{Number of observations with missing data}
#'     \item{ess}{Expected sample size}
#'     \item{opt}{Information from `optim` from weight calculation}
#'     \item{boot}{Parameters and bootstrap sample weights, `NULL` in this object}
#'   }
#' @keywords dataset
#' @family anchored datasets
"weighted_twt"


?? like this?

Reply:
can you answer this too? :) @gravesti

Reply:
I made a suggestion in another comment now